lib/zstd.h

```diff
@@ -1142,11 +1142,16 @@ ZSTDLIB_API size_t ZSTD_CCtx_refPrefix_advanced(ZSTD_CCtx* cctx, const void* pre
  *  Useful after an error, or to interrupt an ongoing compression job and start a new one.
  *  Any internal data not yet flushed is cancelled.
  *  Dictionary (if any) is dropped.
```

Comment:
nit : this comment line is no longer valid.
Also : mention that next compression will use same parameters.

Reply:
Oops, I meant to update the documentation, but completely forgot.